src/widget/scroll.rs

```diff
@@ -294,70 +283,80 @@ impl<A> State<A>
 
                             // Determine the new `Interaction` between the mouse and scrollbar.
                             match (is_over_elem, prev_interaction, mouse.left.position) {
-                                (Some(_),    Normal,             Down) => Normal,
-                                (Some(elem), _,                  Up)   => Highlighted(elem),
-                                (Some(_),    Highlighted(_),     Down) |
-                                (_,          Clicked(Handle(_)), Down) => Clicked(Handle(mouse_scalar)),
-                                (_,          Clicked(elem),      Down) => Clicked(elem),
-                                _                                      => Normal,
+                                (Some(_), Normal, Down) => Normal,
+                                (Some(elem), _, Up) => Highlighted(elem),
+                                (Some(_), Highlighted(_), Down) |
+                                (_, Clicked(Handle(_)), Down) => Clicked(Handle(mouse_scalar)),
+                                (_, Clicked(elem), Down) => Clicked(elem),
+                                _ => Normal,
```

Comment:
I think the symmetry that was originally present in the original format is much more readable for these tupled match expressions. Not sure how to stop rustfmt from touching these though.